app/server.js

```diff
@@ -1,12 +1,12 @@
 "use strict";
 
-const http = require("http"),
+const https = require("https"),
   express = require("express");
 
 const ServerFactory = function(config, logger) {
 
   const app = express(),
-    server =  http.createServer(app);
```

Comment:
@stulees it is possible to have a nodejs app running http and https but there is need to create two separate server instance each one with a different port. So I think in our case as we have a 9494 as the only port there is not much to do. Cheers